### DISCUSSION TITLE
绳网1.12.1会不稳定吗？

### DISCUSSION THREAD

**Discussion by tianxianuo** [share121/inter-knot#2170] [其他]
![Screenshot_2024-04-07-21-28-27-499_tv danmaku bili](https://github.com/user-attachments/assets/5cfceb68-8d03-45dc-a20e-2d6d8d5c1478)
不是网络卡，是直接闪退的那种，已经闪退了三次了，机身发热严重，骁龙660都带不动(ÒωÓױ)！！！
![Screenshot_2024-08-15-10-22-29-471_com mmbox xbrowser](https://github.com/user-attachments/assets/c4e2a99e-baea-4c61-b18b-dac76ab8ee73)

**Comment by 147258huan** [comment]
已经不稳定了

**Comment by LuAn-afl** [comment]
我觉得很好啊跟浏览正常网站一样不闪退不发热<img src="https://img-static.mihoyo.com/communityweb/upload/613a2b262af0319edde21587b88a9c6e.png" alt="米游姬-吃瓜" width="75" >

**Comment by FRIKY-c** [comment]
重新装一下插件试试
